website/src/components/views/overview/LinkedOverview.astro

```diff
@@ -0,0 +1,80 @@
+---
+import { defaultBreadcrumbs } from '../../../layouts/Breadcrumbs';
+import ContaineredPageLayout from '../../../layouts/ContaineredPage/ContaineredPageLayout.astro';
+import { getPathogenMegaMenuSections } from '../../../layouts/base/header/getPathogenMegaMenuSections';
+import { PageHeadline } from '../../../styles/containers/PageHeadline';
+import MainMenuItem from '../../MainMenuItem.astro';
+import { iconMapping } from '../../iconCss';
+
+type Props = {
+    organism: keyof ReturnType<typeof getPathogenMegaMenuSections>;
+    linkedOrganismsTitle: string;
+    linkedOrganisms: (keyof ReturnType<typeof getPathogenMegaMenuSections>)[];
+};
+
+const { organism, linkedOrganismsTitle, linkedOrganisms } = Astro.props;
+
+const section = getPathogenMegaMenuSections()[organism];
+
+const allSections = getPathogenMegaMenuSections();
+const subsections = linkedOrganisms.map((key) => allSections[key]);
+---
+
+<ContaineredPageLayout
+    title={section.headline}
+    breadcrumbs={[...defaultBreadcrumbs, { name: section.headline, href: section.href }]}
+>
+    <PageHeadline>{section.headline}</PageHeadline>
+    <div class='group flex flex-wrap justify-center gap-2'>
+        {
+            section.navigationEntries.map((entry) => {
+                return (
+                    <div class={`card card-border w-96 shadow ${section.borderEntryDecoration}`}>
+                        <a href={entry.href} class='card-body h-full'>
+                            <div class='card-title'>
+                                <h2 class={`iconify ${iconMapping[entry.iconType]}`} />
+                                <div>{entry.label}</div>
+                            </div>
+                            <p>{entry.description}</p>
+                            <div class='card-actions justify-end'>
+                                <span class={`btn btn-soft ${section.headlineBackgroundColor}`}>
+                                    Go to {entry.label}
+                                </span>
+                            </div>
+                        </a>
+                    </div>
+                );
+            })
+        }
+    </div>
```

Comment:
This is copied from the `GenericOverview`, isn't it? I'm thinking whether we can reuse that?

I think it would be best to introduce a `<slot/>` in the `GenericOverview`, keep using the `GenericOverview` in the index.astros and render a common component into that slot? Do you have other suggestions?